distributed/worker_metrics.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from prometheus_client import Counter, Gauge, Histogram, Summary
+
+tasks = Gauge(
+    "dask_worker_tasks",
+    "Current number of tasks at worker",
+    ["state"],
+)
+
+compute = Summary(
+    "dask_worker_compute",
+    "Number of tasks computed and total time spent computing tasks since the latest "
+    "worker restart",
+    unit="seconds",
+)
+
+latency = Histogram(
```

Comment:
`Histogram` metrics also embed a Summary; they replace the `_max` metrics and all crick stuff.